README.md

```diff
@@ -36,12 +36,14 @@ make check
 sudo make install
 ```
 
-Then, `go get -u` as usual.
+Then, `go get -u` as usual the following packages:
 
 ```sh
+go get -u github.com/golang/protobuf/protoc-gen-go
```

Comment:
The reason I chose explicit packages is I kept adding whatever the results of "go build" would complain about such as in "cannot find package "google.golang.org/grpc"

I didn't know about "go get .". I'll just simply use that as I tested that it works.